.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: ruby
 rvm:
-  - 2.1.0
+  - 2.4.0
+  - 2.3.3
+  - 2.2.6
+  - 2.1.10
```

Comment:
Sorry, I haven't had enough coffee yet. I already created the PR: #76 (also also #75 for dropping Ruby 1.9.3)